src/elements/select/select.svelte

```diff
@@ -57,6 +57,9 @@ let open = false;
 let navigationIndex = -1;
```

Comment:
Can we revert the select changes here? It might end up conflicting with new fixes in your other PR.